sources/api/migration/migrations/v1.1.3/kubelet-cpu-manager-state/src/main.rs

```diff
@@ -0,0 +1,59 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::{error, migrate, Migration, MigrationData, Result};
+use snafu::ResultExt;
+use std::fs;
+use std::io;
+use std::path::Path;
+use std::process;
+
+const SETTING: &str = "settings.kubernetes.cpu-manager-policy";
+const CPU_MANAGER_POLICY_CHECKPOINT: &str = "/var/lib/kubelet/cpu_manager_state";
+/// We remove cpu manager policy checkpoint value if checkpoint value is different from upgrade version
+/// or downgrade version cpu-manager-policy setting value.
+pub struct CpuManagerPolicyCleaner;
+
+impl Migration for CpuManagerPolicyCleaner {
+    fn forward(&mut self, input: MigrationData) -> Result<MigrationData> {
+        println!("CpuManagerPolicyCleaner forward has no work to do on upgrade.");
```

Comment:
I think it's best to explain why it doesn't have work to do on upgrade. We're not changing the value of a setting or anything. It's just that we're adding a behavior where we always remove the `cpu_manager_state` checkpoint file, therefore we don't need to explicitly remove the file during forward migration.